jck/runtime.vm/playlist.xml

```diff
@@ -74,7 +74,34 @@
 		<groups>
 			<group>jck</group>
 		</groups>
+                <impls>
+                        <impl>hotspot</impl>
+                </impls>
 	</test>
+        <test>
+                <testCaseName>jck-runtime-vm-jvmti-openj9</testCaseName>
+                <variations>
+                        <variation>NoOptions</variation>
+                </variations>
+                <command>export JAVA_HOME=$(TEST_JDK_HOME)$(D)  $(AND_IF_SUCCESS) \
+        perl $(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+        -test-root=$(Q)$(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)stf;$(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)openjdk-systemtest$(Q) \
+        -systemtest-prereqs=$(Q)$(SYSTEMTEST_RESROOT)$(D)systemtest_prereqs;$(JCK_ROOT)$(Q) \
+        -java-args-setup=$(Q)$(JVM_OPTIONS) -Dcom.ibm.tools.attach.enable=yes$(Q) \
+        -results-root=$(REPORTDIR)  \
+        -test=Jck -test-args=$(Q)tests=vm/jvmti,jckRoot=$(JCK_ROOT),jckversion=$(JCK_VERSION),testsuite=RUNTIME$(Q); \
+        $(TEST_STATUS)</command>
+                <levels>
+                        <level>sanity</level>
+                </levels>
+                <groups>
+                        <group>jck</group>
+                </groups>
+                <impls>
+                        <impl>openj9</impl>
```

Comment:
We run JCKs for only OpenJ9 and IBM sdks anyway, so adding these <impl> tags here seems a bit redundant.

Reply:
@ShelleyLambert suggested them?
In theory someone else could try and run JCK for Hotspot if they were allowed...